client/src/components/music/SpotifyLogin.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import NavBar from "../NavBar";
+import { constants, errors, routes } from "../../Common";
+import axios from "axios";
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles({
```

Comment:
hmmm why are we doing inline styling here instead of a separate file

Reply:
Will change it once spotify is ready dw